code/config/test/Test/Loot/Config.hs

```diff
@@ -61,6 +64,18 @@ type Sub3Fields = '[ "int4" ::: Int ]
 cfg :: PartialConfig Fields
 cfg = mempty
 
+fullConfig :: ConfigRec 'Partial Fields
```

Comment:
We define all types and constant function at the beginning of file
Also, it's easily to read whole file, you start from top and see all definitions, then go forward and see tests

Reply:
Ah, I left this comment because I figured that if unnecessary this would have just made a `git blame` more complex.

However I see you made minor adjustments and the motivation for common functions is good enough for me :ok_hand: